src/http/auth_client.rs

```diff
@@ -46,19 +47,19 @@ impl AuthClient {
         }).unwrap_or_else(|_| HyperClient::with_connector(HttpsConnector::new(TlsClient::default())));
 
         client.set_redirect_policy(RedirectPolicy::FollowNone);
-
-        AuthClient {
-            auth:   auth,
-            client: client,
-        }
+        AuthClient { auth: auth, client: client, version: version }
     }
 
     fn send(&self, req: AuthRequest) -> Response {
         let started = time::precise_time_ns();
+        let mut headers = req.headers.clone();
+        if let Some(ref version) = self.version {
+            headers.append_raw("x-sota-client-version", version.as_bytes().to_vec());
```

Comment:
Shouldn't this just be `User-Agent` like this: https://github.com/heartsucker/rust-tuf/blob/develop/src/repository.rs#L311-L315